nodejs/examples/metrics/mixed/package.json

```diff
@@ -8,8 +8,8 @@
         "build": "tsc"
     },
     "dependencies": {
-        "@pulumi/pulumi": "^1.0.0",
-        "@pulumi/aws": "^2.0.0",
+        "@pulumi/pulumi": "^2.0.0",
```

Comment:
We actually want this to stick to pulumi 1.x -- the point of this test is to check how the potential for SxS pulumi SDK versions works.

Reply:
AH great spot! Thanks for that - fixing up now